src/com/twu/biblioteca/MainMenu.java

```diff
@@ -1,40 +1,62 @@
 package com.twu.biblioteca;
 
+import java.util.ArrayList;
+import java.util.Arrays;
 import java.util.Scanner;
 
 public class MainMenu {
 
-    public void welcome() {
-        System.out.println("Welcome to Biblioteca. Your one-stop-shop for great book titles in Bangalore!");
+    private final String welcome = "Welcome to Biblioteca. Your one-stop-shop for great book titles in Bangalore!";
+    private final String invalidMessage = "Please select a valid option!";
+    private final String goodBye = "Goodbye!";
+
+   ArrayList<String> books = new ArrayList<String>(Arrays.asList(
+           "1. Clean Code | Robert C. Martin | 2008",
+           "2. Don't Make Me Think | Steve Krug | 2000",
+           "3. Test Driven Development | Kent Beck | 2000"));
+
+   ArrayList<String> options = new ArrayList<String>(Arrays.asList(
+           "Please select an option from the following:",
+            "1. List of books",
+            "2. Quit"));
+
+    public void showWelcomeMessage() {
+        System.out.println(welcome);
+    }
+
+    public void showInvalidMessage() {
+        System.out.println(invalidMessage);
+    }
+
+    public void showGoodbyeMessage() {
+        System.out.println(goodBye);
     }
 
     public void showOptions() {
-        System.out.println("Please select an option from the following:");
-        System.out.println("1. List of books");
-        System.out.println("2. Quit");
+        for(int i = 0; i < options.size(); i++) {
```

Comment:
This refactor should payoff when you introduce more options to the menu in the coming stories.